LidarrAPI/Startup.cs

```diff
@@ -26,7 +26,15 @@ public Startup(IConfiguration configuration, IHostingEnvironment env)
             ConfigLidarr = Config.GetSection("Lidarr").Get<Config>();
 
             env.ConfigureNLog("nlog.config");
-            
+
+            Console.WriteLine($"Environment: \nDataDirectory: {Environment.GetEnvironmentVariable("DataDirectory")}\nDatabase     : {Environment.GetEnvironmentVariable("Database")}\n\n");
+
+            // If env variables exist, read those in instead
+            ConfigLidarr.Database = Environment.GetEnvironmentVariable("Database") ?? ConfigLidarr.Database;
```

Comment:
Doesn't the config code (maybe `Config.cs`) already read in from the environment?

Reply:
When I was running it, via the docker-compose, it was NOT reading the environment variables, which is why I added them.. it was still reading the appconfig.json.

However, if needed, we can remove that hunk.

Reply:
Actually nevermind. I'm not seeing those in the config code now. Maybe it was another project I was looking at.